### PR TITLE
Trade using atomic outcome tokens

### PR DESCRIPTION
**Pull request by InfiniteStyles**
Version 1.0

### CODE REVIEW

File: contracts/MarketMaker.sol [hunk 1]
```diff
@@ -174,12 +188,12 @@ contract MarketMaker is Ownable, IERC1155TokenReceiver {
                 collateralToken.approve(address(pmSystem), uint(outcomeTokenNetCost))
             );
 
-            pmSystem.splitPosition(collateralToken, bytes32(0), conditionId, partition, uint(outcomeTokenNetCost));
+            splitPositionThroughAllConditions(uint(outcomeTokenNetCost), conditionIds.length, 0);
```
Comment (cag) [inline contracts/MarketMaker.sol]: I think splitting should also be affected by the `payoutAtomicTokens` parameter in this case. Also, the frontend is not coded to take advantage of this parameter yet.

**Comment by cag** [comment]
Also, I don't feel things are stable enough for us to call this a 1.0 release yet. For example, the ERC1155 draft standard has been un-last-called and may change the core monolith a bit.

File: contracts/MarketMaker.sol (hunk 1 above)
Comment (InfiniteStyles) [reply contracts/MarketMaker.sol]: I've thought about that. I cant quite picture how you envision that working? As, splitting already gets paid in `AtomicOutcomeTokens`.

**Comment by InfiniteStyles** [comment]
> Also, I don't feel things are stable enough for us to call this a 1.0 release yet. For example, the ERC1155 draft standard has been un-last-called and may change the core monolith a bit.

Hmm, this is my bad. I needed to have consulted with you before publishing this as a 1.0 release. It felt right with all the tests passing and the feature list complete to publish it as `1.0` . Unfortunately it is now on NPM and hard to remove, with the new rules. I will look into rewinding it tomorrow.

**Comment by cag** [comment]
:shrug: It's just a number, no big deal. I made that comment before I noticed it was on NPM already. We'll just use semantic versioning as it is expected now, and this will be one of the only 1.x releases of this, as the cat is out of the bag, and I anticipate even more breaking changes to come soon. `yargs` is up to 13.x now :P

BTW, I would like changes to the versioning to be committed to the repository so we have a reference to what was intended to be published on NPM.

As for how to deal with handling both splitting and merging, I have an idea: directly modify the trade array if that boolean is set, subtracting the net cost from every element of the trade array. Then there would be no need to perform a second transfer on each atomic outcome slot. Since this will be a breaking change in intended behavior, the major version number should be incremented.

**Comment by cag** [comment]
As discussed in the meeting today, this change is currently not required, so I'm closing this.

**Pull request closed without merging**